logic/data.py

```diff
@@ -53,6 +53,23 @@ def __init__(self, X: pd.DataFrame, y: NDArray):
             time_to_event=time_to_event,
         )
 
+    def drop_negative_and_zero_timepoints(self) -> int:
```

Comment:
Besides dropping the negative and zero time points, we also wanted to add functionality to the config file to decide the event value and censored value. Could you add this check @zsewa?

Reply:
I will open another PR for this momentarily.